detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnnecessaryApplySpec.kt

```diff
@@ -8,33 +8,60 @@ import org.jetbrains.spek.api.dsl.it
 import org.jetbrains.spek.subject.SubjectSpek
 
 class UnnecessaryApplySpec : SubjectSpek<UnnecessaryApply>({
+
 	subject { UnnecessaryApply(Config.empty) }
 
-	given("some code using apply expressions extensively") {
-		it("reports unnecessary applies that can be changed to ordinary method call") {
-			val findings = subject.lint("""
-				fun b(i : Int) {
-				}
+	given("unnecessary applies that can be changed to ordinary method call") {
+
+		it("reports an apply on non-nullable type") {
+			assertThat(subject.lint("""
 				fun f() {
 					val a : Int? = null
```

Comment:
I guess this should be non-nullable?

```suggestion
					val a : Int = null
```